website/docs/tutorials/basic/running_your_app/6_tab_completion.md

```diff
@@ -19,4 +19,10 @@ Get the exact command to install the completion from `--hydra-help`.
 Currently, Bash and Fish are supported. We are relying on the community to implement completion plugins for additional shells.
 
 Fish support requires version >= 3.1.2.
-Previous versions will work but add an extra space after `.`.
\ No newline at end of file
+Previous versions will work but add an extra space after `.`.
+
+Zsh users can use the existing Bash completions by appending 
+```
+autoload bashcompinit && bashcompinit
+```
+to their `.zshrc` file, restarting their shell and then using the commands provided for Bash.
```

Comment:
cool, so add a title to this section to get something you can link to directly.
by the way - you can run the website locally to see what it looks like before you push.
you can also see the website preview in the push (look for "netlify" in the pr checks).

Reply:
Added a title and added link to `--hydra-help`. Hope the wording is fine.